docs/pages/release_notes_old.md

```diff
@@ -3,8 +3,7 @@ title: Old Release Notes
 permalink: pmd_release_notes_old.html
 ---
 
-Previous versions of PMD can be downloaded here:
-http://sourceforge.net/projects/pmd/files/pmd/
+Previous versions of PMD can be downloaded here: https://github.com/pmd/pmd/releases
```

Comment:
The full archive is only on sourceforge. On github, the releases go only back to since we use github. But I think, that's fair enough. The relevant releases are on github.